package.json

```diff
@@ -4,20 +4,27 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
+    "concurrently": "^9.1.2",
     "typescript": "^5.7.3",
     "vite": "^6.1.0"
   },
   "scripts": {
-    "build:server": "vite build --config vite.server.config.ts",
-    "build:client": "vite build --config vite.client.config.ts",
-    "build": "npm run build:server && npm run build:client"
+    "dep:install": "yarn install && cd src/nui && yarn install",
+    "build:all": "node build.js",
```

Comment:
Isn't there a script missing for build.js and copy-assets.js?

Reply:
watch-assets.js needs to be renamed to watch-assets.mjs.
As we currently do not use type: "module"